lib/stream.js

```diff
@@ -100,6 +104,21 @@ Stream.finished = eos;
 Stream.destroy = destroyer;
 Stream.compose = compose;
 
+const internalSymbols = {
+  kIsDestroyed: utils.kIsDestroyed,
+  kIsDisturbed: utils.kIsDisturbed,
+  kIsErrored: utils.kIsErrored,
+  kIsReadable: utils.kIsReadable,
+  kIsWritable: utils.kIsWritable,
+};
+
+ObjectDefineProperty(Stream, '_internalSymbols', {
+  enumerable: false,
```

Comment:
FWIW `false` is the default value. We should probably make it configurable to limit how breaking this can be if someone is setting `Stream._internalSymbols` in their code.
```suggestion
  __proto__: null,
  configurable: true,
```